### PR TITLE
Migrate script for converting monitors

### PR DESCRIPTION
**Pull request by alapidas**
This migrate script will migrate a non-CC-backed hub + collector toe be
CC-backed.  To be used during the 4x -> 5x migration

### CODE REVIEW

File: Products/ZenModel/migrate/migrateLocalhostPerf.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+class MigrateLocalhostPerf(Migrate.Step):
+    version = Migrate.Version(5, 0, 70)
+
+    def cutover(self, dmd):
+        try:
+            from ZenPacks.zenoss.DistributedCollector import \
+                addHubRoot, addLocalhostHub, convertLocalhostHubToCC, convertAllMonitors, \
+                convertMonitor, convertLocalhostMonitorToCC
+        except ImportError:
+            log.info("Required distributed collector module not found, skipping")
```
Comment (jhanson) [inline Products/ZenModel/migrate/migrateLocalhostPerf.py]: I wouldn't say "Required" here, just silently skip the migrate

File: Products/ZenModel/migrate/migrateLocalhostPerf.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2011, all rights reserved.
```
Comment (jhanson) [inline Products/ZenModel/migrate/migrateLocalhostPerf.py]: Update the copyright

File: Products/ZenModel/migrate/migrateLocalhostPerf.py (hunk 1 above)
Comment (alapidas) [inline Products/ZenModel/migrate/migrateLocalhostPerf.py]: How about
```
log.info("Distributed collector module not found, skipping")
```

Comment (jhanson) [inline Products/ZenModel/migrate/migrateLocalhostPerf.py]: Sure

**Pull request merged**